src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/H2Engine.java

```diff
@@ -473,23 +471,34 @@ protected MappedEntity createPreparedStatementForInserts(final DbEntity entity)
         logger.trace(statement);
 
 
-        final PreparedStatement ps, psReturn, psWithAutoInc, psMerge;
+        PreparedStatement ps = null, psReturn = null, psWithAutoInc = null, psMerge;
```

Comment:
Same question here.

Also, if this PR is to exetnd DB2 and MySql, why are we touching H2 and Pgsql classes?

Reply:
Cool!